src/config/dark-sites.config

```diff
@@ -1,5 +1,3 @@
-*.iaas.store
```

Comment:
It seems you added a single site to the list, but many records moved around. Please check.